doc/user/reporting-performance-problems.md

```diff
@@ -20,6 +20,15 @@ that is unique to us - it applies to many sophisticated virtual machines - but
 most Ruby implementations are not yet doing optimisations powerful enough to
 show them so it may be a new problem to some people in the Ruby community.
 
+## How to check for basic performance problems
+
+If you are examining the performance of TruffleRuby, we would recommend that you
+always run with the `-J-Dgraal.TraceTruffleCompilation=true` flag. If you see
+compilation failures or repeated compilation or the same methods, this is an
```

Comment:
"...repeated compilation or the same methods..." -> "...repeated compilation of the same methods..."

Reply:
Fixed.